WordPress/Classes/Extensions/Colors and Styles/WPStyleGuide+ApplicationStyles.swift

```diff
@@ -108,6 +108,17 @@ extension WPStyleGuide {
                                                    for: .disabled)
 
     }
+
+    class func configureToolbarAppearance() {
```

Comment:
I added this as a global fallback default, in case other areas were affected and we hadn't caught them yet. I don't have a great argument for keeping this, but it felt safe.

Reply:
I agree this is OK to call globally. I also checked other places in the app for regressions and found none.